application.js

```diff
@@ -226,6 +226,11 @@ require(["domReady!", "stix2viz/stix2viz/stix2viz"], function (document, stix2vi
         } else if (xhttp.status != 200 && xhttp.status != 0) {
           alert("ERROR: " + xhttp.status + ": " + xhttp.statusText + " - Double check url provided");
         }
+
+        xhttp.onerror = function() {
+          alert("ERROR: No 'Access-Control-Allow-Origin' header is present on the requested resource. \
+\nA GitHub Gist can be created to host RAW JSON data to prevent this.");
+        };
```

Comment:
This error will be raised/shown if there is _any_ error with the request fetching JSON, even if the error isn't a CORS issue. For instance, I got the message when trying it on a fake website that doesn't exist. Maybe the message should read more like "Unable to fetch that JSON. If you think it should have worked, the requested resource may not have a '...' header. A github...."

Reply:
You're right, I didn't take into account if the website doesn't exist at all. It also looks like the `return` I had in https://github.com/oasis-open/cti-stix-visualization/pull/24/commits/9fee3dfde865d3e657cfa32c692317918ce6cc96 taken out by mistake when resolving the merge conflict.

I'll update the error message. Unfortunatly, there is no way to grab the exact error that was thrown. [Unless the user is using an older version of IE](https://www.html5rocks.com/en/tutorials/cors/)

Thanks for the review! I have to get out of the habit of merging code in before someone looks at it.